src/components/MediaHeaderCtrl/MediaHeaderCtrl.tsx

```diff
@@ -56,7 +56,7 @@ class MediaHeaderCtrl extends Component<Props, State> {
         userSetInfo({ subscribedPodcastIds: response.data })
       }
     } catch (error) {
-      if (error && error.response && error.response.data && error.response.data.message === 'Premium Membership Required') {
+      if (error && error.response && error.response.data && error.response.data.message === PV.components.MediaHeaderCtrl.PremiumMembershipRequired) {
```

Comment:
I'm thinking maybe we should get rid of the components constant entirely and put them in core...components is a folder in the code architecture, but there's no reason to think constants needed in the components directory won't be needed elsewhere in the app, so I'm thinking mixing things in core would at least be better since it's not named after a directory.